src/pages/Creatures.tsx

```diff
@@ -130,24 +94,14 @@ export default function Creatures() {
                 sx={{ paddingTop: "10px", paddingBottom: "10px" }}
               >
                 <FilterButtons
-                  hasFilters={query.filters.length ? true : false}
-                  setIsFilterDrawerOpen={setIsFilterDrawerOpen}
-                  clearFilters={clearFilters}
+                {...filterButtonProps}
+                />
+                <SearchInput 
```

Comment:
Lets still pass the specific props here so we don't couple the hook to the component too much.

You can make code so DRY it becomes brittle.